packages/manager/modules/anthos/src/index.js

```diff
@@ -11,15 +11,15 @@ angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
       template: '<div data-ui-view></div>',
       redirectTo: 'anthos.index',
       resolve: {
-        breadcrumb: () => 'anthos',
+        breadcrumb: () => 'Hosted Private Cloud',
```

Comment:
Should be `Anthos` and should come from the translations file

Reply:
From mockup we have:
Hosted Private Cloud -> Anthos.

So into the root module we must have "**Hosted Private Cloud**" not Anthos.